activemodel/lib/active_model/callbacks.rb

```diff
@@ -103,6 +103,7 @@ def self.extended(base) #:nodoc:
     def define_model_callbacks(*callbacks)
       options = callbacks.extract_options!
       options = {
+        terminator: default_false_terminator,
```

Comment:
I have two suggestions:

1. rename this method to `deprecated_false_terminator`, so its intent is clearer
2. pass as an argument the current module, that is, `terminator: deprecated_false_terminator(ActiveModel)`

I'll show in the next comment why I think 2. is a good idea